src/main/java/frc/robot/Constants.java

```diff
@@ -1,15 +1,126 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
-public final class Constants {}
+public final class CONSTANTS {
+
+    //General Constants
+    public static final int PCM_ID = 3;
+
+    // Subsystem Constants
+    public static final class DRIVE {
+        public static final int FORWARD_CHANNEL_ID = 2;
+        public static final int REVERSE_CHANNEL_ID = 3;
+        public static final int RIGHT_MASTER_ID = 4;
+        public static final int RIGHT_FOLLOWER_1_ID = 5;
+        public static final int RIGHT_FOLLOWER_2_ID = 6;
+        public static final int LEFT_MASTER_ID = 7;
+        public static final int LEFT_FOLLOWER_1_ID = 8;
+        public static final int LEFT_FOLLOWER_2_ID = 9;
+        public static final double SPEED_MULTIPLIER = 0.6;
+        public static final double TURN_MULTIPLIER = 0.4;
+    }
+
+    public static final class STORAGE {
+        public static final int BALLS_INTAKE_SWITCH_ID = 0;
+        public static final int BALLS_IN_SWITCH_ID = 1;
+        public static final int BALLS_OUT_SWITCH_ID = 2;
+        public static final int POLYCORD_MOTOR_ID = 11;
+        public static final double POLYCORD_FORWARD_SPEED = .3;
+        public static final double POLYCORD_REVERSE_SPEED = -.2;
+    }
```

Comment:
Can you clean up the parts that aren't being used?

I think the controller ones are fine, but we should try to trim this.